app/models/main.py

```diff
@@ -232,6 +238,16 @@ def validates_clarification_questions_open(self, key, value):
     def __repr__(self):
         return '<{}: {} slug={}>'.format(self.__class__.__name__, self.name, self.slug)
 
+    @validates('has_direct_award', 'has_further_competition')
+    def validates_framework_components(self, key, value):
+        has_direct_award = value if key == 'has_direct_award' else self.has_direct_award
+        has_further_competition = value if key == 'has_further_competition' else self.has_further_competition
+
+        if has_direct_award is False and has_further_competition is False:
```

Comment:
The trouble with using `is False` here, is that `None` here is getting treated as `True`. Which - at database layer anyway - we aren't enforcing.

Reply:
This is required in order to set any values whatsoever, as otherwise the moment we update one of the values, validation will fail because the other is still None. We don't even technically need this validation as it's just an interim check that will last all of a couple of days.